simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/StationState.java

```diff
@@ -160,4 +163,19 @@ public String toString() {
                         .filter(connector -> connector.getId().equals(connectorId))
                         .findAny());
     }
+
+    public void refreshView() {
```

Comment:
I don't understand the purpose of this method... can you not just simply return copyOf(this) inside getView.
Also it's kind of weird for a class to have a member of the same type.